cmd/watch.go

```diff
@@ -2,6 +2,7 @@ package cmd
 
 import (
 	"fmt"
+	util2 "github.com/redhat-developer/odo/pkg/odo/util"
```

Comment:
why util2? should we have this renamed instead?

Reply:
Sure, I could rename this, this was done automatically…

Reply:
Fixed.